client/src/featureform/serving.py

```diff
@@ -180,6 +190,12 @@ def features(
 
         return feature_values
 
+    def get_source_as_df(self, name, variant):
```

Comment:
was this the agreed upon naming?

Reply:
No, and now that I'm thinking about it, I should have added an underscore prefix to the method name.

My thought here was to define a method that was shared across both implementations that would be wrapped by the "public" `dataframe` method we're defining on `Client`. Eventually, when we refactor, the implementation will be added to `Client` and we can work with a standard API for both hosted and local mode.

Reply:
I thought the functionality would be on the source directly? `average_user_transaction.df()`

Reply:
No it's via the Client now

Reply:
I think method name is ok